# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../ghostscript/no_gtk.diff
# Copyright (C) 2007 The OpenSDE Project
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Only build the Gtk+ binary when gtk-12 is around.

--- ./src/unix-dll.mak.orig	2006-09-28 13:54:45.000000000 -0400
+++ ./src/unix-dll.mak	2007-03-17 19:15:57.000000000 -0400
@@ -87,7 +87,7 @@
 
 # Normal shared object
 so: SODIRS
-	$(MAKE) $(SODEFS) CFLAGS='$(CFLAGS_STANDARD) $(CFLAGS_SO) $(ACDEFS) $(GCFLAGS) $(XCFLAGS)' prefix=$(prefix) $(GSSOC) $(GSSOX)
+	$(MAKE) $(SODEFS) CFLAGS='$(CFLAGS_STANDARD) $(CFLAGS_SO) $(ACDEFS) $(GCFLAGS) $(XCFLAGS)' prefix=$(prefix) $(GSSOC)
 
 # Debug shared object
 # Note that this is in the same directory as the normal shared
@@ -103,7 +103,6 @@
 	-mkdir -p $(install_prefix)$(bindir)
 	-mkdir -p $(install_prefix)$(libdir)
 	$(INSTALL_PROGRAM) $(GSSOC) $(install_prefix)$(bindir)/$(GSSOC_XENAME)
-	$(INSTALL_PROGRAM) $(GSSOX) $(install_prefix)$(bindir)/$(GSSOX_XENAME)
 	$(INSTALL_PROGRAM) $(BINDIR)/$(SOBINRELDIR)/$(GS_SONAME_MAJOR_MINOR) $(install_prefix)$(libdir)/$(GS_SONAME_MAJOR_MINOR)
 	$(RM_) $(install_prefix)$(libdir)/$(GS_SONAME)
 	ln -s $(GS_SONAME_MAJOR_MINOR) $(install_prefix)$(libdir)/$(GS_SONAME)
